test/integration/src/client-util.js

```diff
@@ -0,0 +1,10 @@
+const configUtil = require('./config-util');
+const revai = require('revai-node-sdk');
+
+module.exports = {
+    getClient: function (apiKey) {
```

Comment:
we could use the arrow syntax

```suggestion
    getClient: (apiKey) => {
```